route_builder.go

```diff
@@ -353,8 +353,25 @@ func (b *RouteBuilder) Build() Route {
 	return route
 }
 
+type MergePathStrategyFunc func(path1, path2 string) string
+
+var (
+	// behavior 3.10.*
+	PathJoinStrategy = path.Join
+
+	// behavior <= 3.9
+	TrimSlashStrategy = func(path1, path2 string) string {
+		return strings.TrimRight(path1, "/") + "/" + strings.TrimLeft(path2, "/")
+	}
+
+	// MergePathStrategy is the active strategy for merging a Route path when building the routing of all WebServices.
+	// The value is set to TrimSlashStrategy
+	// PathJoinStrategy is an alternative strategy that is more strict [Security - PRISMA-2022-0227]
+	MergePathStrategy = TrimSlashStrategy
```

Comment:
I think you need to set it to `PathJoinStrategy` as default

Reply:
yes, agree. Reason for it is because that will fix the security issue that motivated the change (which turned out to be a breaking one).